client/asset/eth/eth.go

```diff
@@ -935,16 +952,21 @@ func (eth *ExchangeWallet) LocktimeExpired(contract dex.Bytes) (bool, time.Time,
 		return false, time.Time{}, err
 	}
 
-	header, err := eth.node.bestHeader(eth.ctx)
+	swap, err := eth.node.swap(eth.ctx, secretHash, contractVer)
 	if err != nil {
 		return false, time.Time{}, err
 	}
-	blockTime := time.Unix(int64(header.Time), 0)
 
-	swap, err := eth.node.swap(eth.ctx, secretHash, contractVer)
+	// Time is not yet set for uninitiated swaps.
+	if swap.State == dexeth.SSNone {
+		return false, time.Time{}, asset.ErrSwapNotInitiated
+	}
```

Comment:
Useful for not logging this as an error, as it is expected until the init is mined.